### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kricheso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,6 +342,36 @@ async function myFunction() {
 }
 ```
 
+#### <a name="getGridForUser" />`Firestore.get.gridForUser(userId, gridId)`
+Gets the grid object for a logged in user. The userId corresponds to the user that is currently logged in. The gridId is the requested grid object. Note: all Firestore functions do this automatically.
+* <b>userId</b> `String` - The id of the logged in user.
```
Comment (Azeezah) [inline README.md:347]: Maybe we can state explicitly somewhere that these are user email addresses?  (Although that's clarified by the examples.)

Comment (Azeezah) [reply README.md:347]: Oh I guess that's shown enough by example.

**Review by Azeezah — APPROVED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (kricheso) [reply README.md:347]: Resolved and merged.

**Pull request merged**